openff/bespokefit/executor/services/qcgenerator/worker.py

```diff
@@ -21,14 +21,16 @@
 from qcelemental.util import serialize
 from qcengine.config import get_global
 
-from openff.bespokefit.executor.services import settings
+from openff.bespokefit.executor.services import current_settings
 from openff.bespokefit.executor.utilities.celery import configure_celery_app
 from openff.bespokefit.schema.tasks import OptimizationTask, Torsion1DTask
 
+__settings = current_settings()
+
 redis_connection = redis.Redis(
-    host=settings.BEFLOW_REDIS_ADDRESS,
-    port=settings.BEFLOW_REDIS_PORT,
-    db=settings.BEFLOW_REDIS_DB,
+    host=__settings.BEFLOW_REDIS_ADDRESS,
+    port=__settings.BEFLOW_REDIS_PORT,
+    db=__settings.BEFLOW_REDIS_DB,
 )
 celery_app = configure_celery_app("qcgenerator", redis_connection)
```

Comment:
The info in the `redis_connection` object seems to be the same as that in the settings class and this logic seems to be repeated for each app, would it make sense to have the `configure_celery_app` just call the `current_settings` function and get the relavent info?

Reply:
Yeah I think either having configure celery return a tuple of both the app and redis connection, or adding a new `configure_redis` method that also checks for the executor key you mentioned in slack / any other validation would be a great shout! 

I think I have a slight preference for the latter, but not by much 🙂

Reply:
We should probably handle this in a new PR though so it doesn't get buried here! 😅